fields/types/gcsimage/GcsImageColumn.js

```diff
@@ -8,11 +8,28 @@ const _ = {
 var GcsImageColumn = React.createClass({
 	render: function () {
 		var value = this.props.data.fields[this.props.col.path] || {};
-		var isVal = _.get(value, ['resizedTargets', 'mobile', 'url'], value.url);
+		const thumbnail = _.get(value, ['resizedTargets', 'mobile', 'url']);
+    const gcsDir = _.get(value, 'gcsDir', '');
+    const gcsBucket = _.get(value, 'gcsBucket', '');
+    const filename = _.get(value, 'filename', '');
+    const location = gcsDir ? `gs://${gcsBucket}/${gcsDir}/${filename}` : `gs://${gcsBucket}/${filename}`;
+
+    const displayJsx = thumbnail ? (
+      <a href={thumbnail} target="_blank">
+        <img src={thumbnail} height="150" width="150" />
+      </a>
+    ) : (
+      <p>
+        {location}
```

Comment:
Minor.
We could still use the [signed url](https://cloud.google.com/storage/docs/access-control/signed-urls) here for displaying raw photo as fallback. This could provide the access with a limited period of time.